### PR TITLE
chore(client): cleanup demo

### PR DESCRIPTION
**Pull request by momentiris**
This is for getting the application in a state where it is somewhat presentable. Not all changes are necessarily definite but it gets us into a state presenting the functionality we have right now.

Doesn't close any issue since there isn't one for this

### CODE REVIEW

File: packages/client/bsconfig.json [hunk 1]
```diff
@@ -7,10 +7,6 @@
     {
       "dir": "src",
       "subdirs": true
-    },
-    {
-      "dir": "__tests__",
-      "type": "dev"
```
Comment (lessp) [inline packages/client/bsconfig.json]: Can remove this now since unit-tests are merged and working again!

EDIT: Or rather, undo these changes

File: packages/client/src/components/Input.re [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        ~inputRef=React.useRef(Js.Nullable.null)->ReactDOMRe.Ref.domRef,
+        ~onChange=?,
+        ~placeholder,
+        ~id,
+        ~label,
+        ~type_="text",
+      ) => {
+    <div className="mb-4">
+      <label htmlFor=id className="block text-gray-700 text-sm font-bold mb-2">
+        label->React.string
```
Comment (lessp) [inline packages/client/src/components/Input.re]: We could lock this to be of type `I18n.Translations.t` if we'd like?

```suggestion
        {I18n.Translations.(toString(label))->React.string}
```

Comment (momentiris) [reply packages/client/src/components/Input.re]: Good idea!

File: packages/client/src/App.re [hunk 3]
```diff
@@ -18,7 +20,7 @@ let make = () => {
       | ["boka-resa"] => <Book />
       | ["resor"] => <Trips />
       | ["resa", id] => <Trip id />
-      | _ => <p> {I18n.Error.toString(FourOFour)->React.string} </p>
+      | _ => <Typography.Error> {errorToString(FourOFour)} </Typography.Error>
```
Comment (lessp) [inline packages/client/src/App.re]: I like this! I wonder if we could take it one step further and only pass it a variant? `FourOFour`, it would save us from having to `LocaleContext.use` everywhere. Might be something for another ticket!

Comment (momentiris) [reply packages/client/src/App.re]: Sweet lets do it. I'll have a look while I'm solving merge conflicts

Comment (momentiris) [reply packages/client/src/App.re]: Lets save this for later. Need to think it through a bit more.
